indexer/services/ender/src/lib/types.ts

```diff
@@ -188,14 +189,26 @@ export interface SingleTradeMessage extends TradeMessage {
   eventIndex: number,
 }
 
+export interface AnnotatedSubaccountMessage extends SubaccountMessage {
+  orderId?: string,
+  isFill?: boolean,
+}
+
+export function convertToSubaccountMessage(
+  annotatedMessage: AnnotatedSubaccountMessage,
+): SubaccountMessage {
+  const subaccountMessage: SubaccountMessage = _.omit(annotatedMessage, ['orderId', 'isFill']);
+  return subaccountMessage;
+}
+
```

Comment:
nit: Is there a helpers file this can go into?

Reply:
moved to helpers